Automattic/wp-calypso
# FSE: Include a page template rather than parsing to render parts.

Include a page template rather than parsing to render parts. This will check first that the post is a `page`, and that the page has post meta of `_wp_template_id` set.

#### Testing instructions

* Switch to this branch, and cherry pick in #34142 .
* Clear all existing templates and parts, then deactivate and reactivate the plugin.
* Create a page, apply the page template to it.
* Verify the header on the front page properly renders the site title and description blocks.

Fixes #34166 .
